site/docs/components/status-indicator/examples.mdx

```diff
@@ -0,0 +1,26 @@
+---
+title: Status Indicator
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+  <LivePreview componentName="status-indicator" exampleName="Status">
+    Use the `status` prop to change the status displayed. This will affect both
```

Comment:
might be a good idea to add title to all examples (aligning it with all other components)

```suggestion
### Status

    Use the `status` prop to change the status displayed. This will affect both
```